src/main/java/racingcar/StringCalculator.java

```diff
@@ -0,0 +1,65 @@
+package racingcar;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    Integer result = 0;
+
+    public Integer add(String input) {
+
+        if (input == null || input.isEmpty()) {
+            return 0;
+        }
+
+        Integer parseInt = isInteger(input);
+        if (parseInt != null) {
+            return parseInt;
+        }
+
+        String delimiter = ",:";
+        String custom = findCustomDelimiter(input, "//(.*?)\\n");
+        if (custom != null) {
+            delimiter += custom;
+            input = input.replaceAll("//(.*?)\\n", "");
+        }
+
+        String[] split = input.split("["+delimiter+"]");
+        for (String s : split) {
+            Integer num = isInteger(s);
+            if (num != null) {
+                result += num;
+            }
+        }
+
+        return result;
+    }
+
+    // 숫자 변환
+    Integer isInteger(String input) {
+        try {
+            Integer num = Integer.parseInt(input);
```

Comment:
해당 값은 굳이 래퍼 타입으로 선언할 필요 없을 것 같아요.

```suggestion
            int num = Integer.parseInt(input);
```

래퍼런스 : https://smin1620.tistory.com/287